lib/index.d.ts

```diff
@@ -259,7 +259,7 @@ export interface VerifyMessageResult {
     errors?: Error[];
 }
 export interface VerifyMessageOptions extends VerifyOptions {
-    detached?: boolean;
+    detached?: boolean; // TODO what is this?
```

Comment:
I think the web app passed this property to verify a detached signature, not realizing that it's unnecessary, and then someone added it here to make TS happy :sweat_smile: